docs/notebook.md

```diff
@@ -110,7 +110,108 @@ We'll walk through two notebook extensions:
 
 ### Adding a button to the toolbar
 
-Coming soon!
+Start from the cookie cutter extension template.
```

Comment:
I think we want to be very explicit and mention that developing extensions only works with a stable release of jupyterlab.